include/branches.csv

```diff
@@ -1,7 +1,7 @@
-Branch,Schedule,Status,First release,End-of-life,Release manager
-main,:pep:`693`,features,*2023-10-02*,*2028-10*,Thomas Wouters
-3.11,:pep:`664`,bugfix,2022-10-24,*2027-10*,Pablo Galindo Salgado
-3.10,:pep:`619`,bugfix,2021-10-04,*2026-10*,Pablo Galindo Salgado
-3.9,:pep:`596`,security,2020-10-05,*2025-10*,Łukasz Langa
-3.8,:pep:`569`,security,2019-10-14,*2024-10*,Łukasz Langa
-3.7,:pep:`537`,security,2018-06-27,*2023-06-27*,Ned Deily
+Branch,Schedule,Status,First release,End-of-life,Release manager

```

Comment:
I'm guessing maybe the EoL changed here? Can you double-check its (now) `\n`?

Reply:
Got confused by end-of-life/end-of-line there for a moment!

It was defaulting to `\r\n`, now updated to `\n`:

https://docs.python.org/3/library/csv.html?highlight=csv#csv.Dialect.lineterminator

Reply:
Oh, haha—should have anticipated that!

Yeah, that triggers my memory on how to do that, thanks—I'd remembered doing it before, but its been a while since I've used `csv` instead of Pandas and browsing through the docs, I couldn't find it—just admonitions to only use `newline=""` in the `open` call.